data_prep/scotland/get_reference_layers.sh

```diff
@@ -80,6 +80,39 @@ function population {
   tippecanoe tmp/population.geojson -zg --generate-ids -l population -o $OUT/population.pmtiles
 }
 
+function stats19 {
+  # Use code from https://github.com/acteng/atip-data-prep/tree/main/stats19
+  git submodule init
+  git submodule update
+
+  cd atip-data-prep/stats19
+  mkdir -p input tmp
+
+  # URLs from https://www.data.gov.uk/dataset/cb7ae6f0-4be6-4935-9277-47e5ce24a11f/road-safety-data
+  if [ ! -e "input/dft-road-casualty-statistics-casualty-1979-latest-published-year.csv" ]; then
+    download_to_subdir input https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-casualty-1979-latest-published-year.csv
+  fi
+  if [ ! -e "input/dft-road-casualty-statistics-collision-1979-latest-published-year.csv" ]; then
+    download_to_subdir input https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-collision-1979-latest-published-year.csv
+  fi
+
+  cargo run --release
+
+  download_to_subdir input https://raw.githubusercontent.com/georgique/world-geojson/refs/heads/develop/areas/united_kingdom/scotland.json
+  mapshaper tmp/stats19.geojson -clip input/scotland.json -o tmp/stats19_clipped.geojson
```

Comment:
With all of the layers in here, I'm imagining to use them for prioritization. we'd also convert to a gpkg or fgb or gmd or other format too.

Reply:
I'm imagining another entry into `data_prep/src/bin/generate-prioritization-layers` (name TBD) that does this.

I'm actually planning to get started on this today with the SIMD dataset. We won't process it yet, but I think in any case we're pretty confident we'll want all the population "zone" polygons in a format accessible to rust and they'll need to be pre-sliced by LAD and REGION... right?

Reply:
Sounds right to me, yep. I think starting this soon and starting to match these to generated nhood boundaries is a good idea, even before the scoring is nailed down. I can almost guarantee it's something we'll iterate on and tune for all cases anyway.